Makefile

```diff
@@ -0,0 +1,17 @@
+.PHONY: help build push
+
+help:
+	@echo ''
+	@echo 'Usage: make [TARGET] [EXTRA_ARGUMENTS]'
+	@echo 'Targets:'
+	@echo '  build    	build docker --image-- '
+	@echo '  push  	push docker --image-- '
+	@echo ''
+	@echo 'Extra arguments:'
+	@echo 'driver-version=:	make driver-version="390.116"'
+
+build:
```

Comment:
I think should be a separate build step. You can use this job -> https://github.com/giantswarm/architect-orb/blob/master/src/jobs/push-to-docker.yaml

Reply:
I leave Makefile for testing purposes on local, I have added the circle CI config